pkg/factories/rune_factory.go

```diff
@@ -242,3 +297,41 @@ func writeKafkaSource(dw printers.PrefixWriter, source *v1alpha1.KafkaSource) {
 	dw.WriteAttribute("Topics", strings.Join(source.Spec.Topics, ","))
 	dw.WriteAttribute("ConsumerGroup", source.Spec.ConsumerGroup)
 }
+
+// printKafkaSource populates a single row of kafka source list table
+func printKafkaSource(kafkaSource *v1alpha1.KafkaSource, options printers.PrintOptions) ([]metav1.TableRow, error) {
```

Comment:
@Kaustubh-pande I think we can safely remove error from return statement here. And below in the `printKafkaSourceList` function as well. It's not actually passing any error message from the code, only `nil` value.

Reply:
@dsimansk we need to add back two Err call, inside tableHandler is validates print handler signature & It must be of the following type:
``func printFunc(object ObjectType, options PrintOptions) ([]metav1beta1.TableRow, error)``